Resources/Views/index.leaf

```diff
@@ -256,6 +257,100 @@
       </table>
     </div> <!-- reference -->
 
+    <div class="best-practices toggle">
+        <h2> The main fallacy to be aware of: <tt>dateFormat</tt> is not locale-aware </h2>
+
+        <p>You should be aware that using a custom <tt>dateFormat</tt> comes with a risk of falling into some fallacies.</p>
+        <p>
+            Especially, you need to realise that the user can use different <tt>Locales</tt>,
```

Comment:
You're going to cringe, but since the rest of the site written from a US Locale, can you change this to `realize`? 😬

Reply:
Fixed in 20c6baf1b8c8780c8fd73bf71bc6d81e595fb933 👍